src/NovelRT/Timing/DeltaTimestamp.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <NovelRT.h>
+
+namespace NovelRT::Timing {
+
+  DeltaTimestamp::DeltaTimestamp(uint64_t tickDelta, double doubleDelta, float floatDelta) : _tickDelta(tickDelta), _doubleDelta(doubleDelta), _floatDelta(floatDelta) {}
```

Comment:
nit: The actual implementation is trivial, I don't see any reason why it needs to be part of the resulting library rather than just in the header file.

Reply:
Because, by convention, weve always put constructors into a source file.

This would become the exception to change that now, no?

Reply:
No, there are a few places where it is pure header, namely around simple types like this.

We probably want to do it for other simple types as well as it will help codegen.

Reply:
Alright I shall move it then. :)